andela/titan-dev-ah-frontend
# 165054461 Configure coveralls

#### What does this PR do?
This PR sets up the Coveralls and sample test
#### Description of Task to be completed?
* We add `--coverage` to the `test` line in the package.json file to trigger the coverage report
#### How should this be manually tested
- Fork the repo and check out the branch `ch-configure-coveralls-#23165054461`
- In the root folder run `yarn` to install all dependencies
- Run `yarn test` to start the project
#### Any background context you want to provide?
None  
#### What are the relevant pivotal tracker stories?
#165054461 
#### Questions:
Non
